js/core/src/registry.ts

```diff
@@ -68,6 +71,18 @@ export class Registry {
   private allPluginsInitialized = false;
 
   readonly asyncStore = new AsyncStore();
+  readonly dotpromptEnv = dotprompt({
+    schemaResolver: async (name) => {
+      const resolvedSchema = await this.lookupSchema(name);
+      if (!resolvedSchema) {
+        throw new GenkitError({
+          message: `Schema '${name}' not found`,
+          status: 'NOT_FOUND',
+        });
+      }
+      return toJsonSchema(resolvedSchema);
+    },
+  });
```

Comment:
Could you switch this to `new DotpromptEnvironment` and import that directly? I'm thinking I want to get rid of `dotprompt` as a toplevel export because I think I want it to be:

```ts
const dotprompt = new DotpromptEnvironment(...);
```

or something like that instead. Otherwise it's hard to know what to name the variable.

Reply:
See https://github.com/google/dotprompt/pull/13 <-- mostly I'm making a breaking change to Dotprompt so let's coordinate before we code freeze.

Reply:
Done.